etna/series/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+
+from wagtail.admin.panels import FieldPanel
+from wagtail.search import index
+
+from wagtailmetadata.utils import get_image_model_string
+
+
+class SeriesCategory(models.TextChoices):
+    RECORD_COLLECTION = "Record collection", "Record collection"
+
+
+class Series(index.Indexed, models.Model):
```

Comment:
**Thoughts:** Is series the right name? Should this be collection? Record series? Does this already exist in a different form?

Reply:
Let's go for `RecordSeries` 👌

Reply:
Perfect.